classes/install/Upgrade.php

```diff
@@ -212,7 +213,74 @@ public function migrateSRLocale()
 
         $oldLocaleStringLength = 's:5';
 
-        $journalSettingsDao = new \PKP\db\DAO();
+        $journalSettingsDao = new class () extends \PKP\db\DAO {
```

Comment:
@asmecher There are several calls to `updateSetting()`/`getSetting()` using the `$journalSettingsDao` below, but the `\PKP\db\DAO` doesn't have such methods.

So I just copied from somewhere and adapted here.

Reply:
This will only affect users who are upgrading from 3.1.x and are using the `sr_SR` locale, which will be a very small demographic, but please go ahead and add this to `main`.